static/bidder-info/audienceNetwork.yaml

```diff
@@ -1,5 +1,5 @@
 maintainer:
-  email: "info@prebid.org"
+  email: "none"
```

Comment:
This is fine. AppNexus/Xandr built and recently updated this adapter, but don't have an official relationship with Facebook. We'll continue to keep an eye on related issues. If Facebook would later like to own it, all the better. :)